mealpy.py

```diff
@@ -27,31 +29,35 @@
 
 KEYRING_SERVICENAME = BASE_DOMAIN
 
-# TODO(GH-4): Replace this feature toggle with cli option
-USE_KEYRING = False
-
 
 def load_config():
     schema = strictyaml.Map({
         'email_address': strictyaml.Email(),
+        'use_keyring': strictyaml.Bool()
     })
-
     root_dir = path.abspath(path.dirname(__file__))
-    with open(path.join(root_dir, 'config.yaml')) as config_file:
+    fname = path.join(root_dir, 'config.yaml')
+    # Create new file using template if not already existing
+    if not path.isfile(fname):
+        template = path.join(root_dir, 'config.template.yaml')
+        copyfile(template, fname)
```

Comment:
This should abort the script and advise user to update config values.

Reply:
One problem with this current approach is if we add new config values into the template, and the user still has an old config file, it will not contain our "default" values as expected. To resolve this we can simply read from the template config file as defaults to fallback on. Kind of like `.vimrc`.